ts/features/walletV3/transaction/components/WalletTransactionInfoSection.tsx

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable functional/immutable-data */
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  Divider,
+  IOColors,
+  IORadiusScale,
+  IOVisualCostants,
+  ListItemHeader,
+  ListItemInfo,
+  ListItemInfoCopy,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import I18n from "../../../../i18n";
+import { Psp, Transaction } from "../../../../types/pagopa";
+import { format } from "../../../../utils/dates";
+import { clipboardSetStringWithFeedback } from "../../../../utils/clipboard";
+import TransactionReceiptDivider from "../../../../../img/features/wallet/transaction-receipt-divider.svg";
+
+type WalletTransactionInfoSectionProps = {
+  transaction?: Transaction | null;
```

Comment:
I've seen this typing procedure of `value?: something|null` around this PR, and I was wondering: why null check? shouldn't the possible `undefined` be enough?

Reply:
It was `Transaction | null` because into `walletTransactionDetailsSelector` selector there was a pot.getOrElse that throwed `null`, thanks for the hint i changed it to `pot.toUndefined` and removed the null references into b3f4a0e2fe60f4738d46d380da5060c9cfe2633f